api/notd/model.py

```diff
@@ -79,6 +80,18 @@ class Token:
     registryAddress: str
     tokenId: str
 
+    def to_dict(self) -> Dict:
+        return{
+            'registryAddress': self.registryAddress,
+            'tokenId': self.tokenId,
+        }
+
+    def from_dict(self: Dict):
```

Comment:
the problem is here, obviously seld cannot be a dict. this should look like this:
```
@classmethod
def from_dict(cls, dict: Dict):
    return cls(registryAddress=dict['registryAddress'], tokenId=self['tokenId'])
```